plone/namedfile/utils/svg_utils.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+from io import BytesIO
+from logging import getLogger
+
+import re
+import xml.etree.cElementTree as et
+
+
+log = getLogger(__name__)
+
+
+def process_svg(data):
+    content_type = None
+    w = -1
+    h = -1
+    size = len(data)
+
+    tag = None
+    try:
+        for event, el in et.iterparse(BytesIO(data), ("start",)):
+            tag = el.tag
+            _w = el.attrib.get("width")
+            _h = el.attrib.get("height")
+            if len(_w) and len(_h):
+                w = int(float(re.sub(r"[^\d\.]", "", _w)))
+                h = int(float(re.sub(r"[^\d\.]", "", _h)))
+            break
```

Comment:
the SVG specification did allow all kinds of width and height values, also values like "100%" or auto. 

There should be a few tests that actually checks if that work.

Reply:
OK... this makes sense. Will add tests.

Reply:
@loechel thanks for the heads up. I also added the tests.